src/utils/getUrlNetworkParams.ts

```diff
@@ -0,0 +1,46 @@
+import { AvaNetwork } from '@/js/AvaNetwork'
+
+export const NETWORK_ALIAS = {
```

Comment:
setting networks as constant there and we have them alredy defined in the init action in network module is not good
if we want to do some modifications it will be hard to track the changes